flink-core/src/main/java/org/apache/flink/configuration/RestOptions.java

```diff
@@ -58,6 +58,15 @@ public class RestOptions {
                                     + " (“50100-50200”) or a combination of both. It is recommended to set a range of ports to avoid"
                                     + " collisions when multiple Rest servers are running on the same machine.");
 
+    /** The url prefix that should be used by clients to construct the full target url. */
+    public static final ConfigOption<String> URL_PREFIX =
+            key("rest.url-prefix")
+                    .stringType()
+                    .defaultValue("/")
+                    .withDescription(
+                            "The url prefix that should be used by clients to construct the full target url, must start and end with '/'."
+                                    + " Attention: This option is respected only if the high-availability configuration is NONE.");
```

Comment:
We can clarify where the prefix will be placed in the rest url. It would be better to give an example.

Reply:
updated